civihr_default_theme/assets/js/civihr_default_theme.script.js

```diff
@@ -44,24 +46,33 @@
    * @param {string} inputFieldSelector
    */
   Drupal.civihr_theme.createDragAndDrop = function (inputFieldSelector) {
+    var dropHelper;
     var inputField = $(inputFieldSelector);
-    var dropHelper = '<span><i class="fa fa-cloud-upload" aria-hidden="true"></i><br>' +
+
+    if (!$('body.mobile').length) {
```

Comment:
Do you think it might be more readable if you extracted `$('body.mobile').length` to a variable like `isMobile`